context-free.md

```diff
@@ -1,22 +1,56 @@
 # Context Free
 
 ##  Who is this programming language for?
-
+The language is targeted toward anyone with access to a computer (Windows,
+Mac, or Linux).  
```

Comment:
I also added that its for people interested in computer-generated art, either for creative reasons or graphic design.